### ISSUE TITLE
Playback through earpiece 

### ISSUE THREAD

**Issue by Mo-hdb44** [TGX-Android/Telegram-X#541]
**Voice messages** **won't** play through my **earbuds** When I choose  **Always** for the **Playback through earpiece** option.

**Comment by matinlunaire** [comment]
This option controls whether audio will be routed to your phone's front speaker (that's typically used for calls), and uses the proximity sensor to detect that the phone's been raised to an ear (if "when close to an ear" is selected). You probably don't want it enabled if you intend to listen to audio messages through earbuds or external speakers.

**Comment by Mo-hdb44** [comment]
yes. I want it to play the voice massages from the front speaker unless I'm using a earbud. it doesn't make sense how it works now.
